src/esphomelib/application.h

```diff
@@ -682,7 +684,16 @@ class Application {
                                              uint32_t update_interval = 15000);
 #endif
 
+#ifdef USE_RTC_COMPONENT
+  time::RTCComponent *make_rtc_component(const std::string &tz = "UTC");
```

Comment:
So do I understand correctly that an RTC component *without* sntp/other time providers just reports the uptime?

Reply:
My understanding is that (on the ESP32 at least) the RTC lives in the ULP. The memory is cleared along with everything else on boot. It is possible to keep the ULP running while the rest of the system (including main memory) is shut down, in which case the main processor registers and main memory are both lost. But yeah, for most cases, the RTC resets to 0 on boot.

There are a number of battery-backed RTCs available; I was thinking that it might be nice to transparently use and NTP sync those instead of the system RTC if one is available, but that might not be easy to do.